### DISCUSSION TITLE
Put your bug found using collaborative bug finding (similar bugs in GitHub) here

### DISCUSSION THREAD

**Discussion by stan6** [SOEN345-WINTER2024/discussion#20] [Announcements]
Refer to https://github.com/orgs/SOEN345-WINTER2024/discussions/3 for example bug report

**Comment by Tharsipa** [comment]
### **Error Handling Issue During Photo Upload with Existing Account**

### **How to reproduce:**

1. Launch the CycleStreets app
2. Go to Dashboard
3. Click “Add Photo”
4. Click “Select existing photo”
5. Click “Signin with an existing account”
6. Input the username and password, then click “Sign In”. You should see a feedback message saying, “You have successfully signed into CycleStreets”.
7. Click “Select existing photo”
8. Select the picture, write a description and click “Next”
9. Input the details of the picture and click “Next” again
10. Select the location of where the picture was taken on the map and click “Upload”.
11. An error message will appear saying, “There was a problem uploading your photo: No validated account matching that username and password was found.” This will cause the app to crash.

### **Actual Behavior:**
<img src="https://github.com/SOEN345-WINTER2024/discussion/assets/90727247/a5ad7cf0-47a0-4ed9-8add-36cf7379a4fa" width="300">

### **Expected Behaviour:**
The expected behavior involves a smooth and error-free process for uploading a photo from an existing account.

**Device Model:** Pixel 7 
**Android Version:** Android 8.1
**App version:** net.cyclestreets/3.10.2/1658

### **Duplicate Check**
I have searched the existing issues and confirm that this bug has not been reported before.

**Link to issue reported:**[issue](https://github.com/cyclestreets/android/issues/550)

**Comment by s-vali** [comment]
## Login bug: Page does not produce appropriate fields for creating an account or logging in

### How to reproduce:
1. Launch Infinity For Reddit app.
2. Click 'burger menu' icon in the top navigation bar. 
3. Click 'Press here to login' button in side menu popup.
4. Click 'Add an account'.
5. Agree to User Agreement. 
6. The login page will be displayed with no fields for entering your credentials. 

### Actual behavior:
![image](https://github.com/SOEN345-WINTER2024/discussion/assets/91510546/95904639-c2a8-4321-a703-9f522d3a8a85)

### Expected behavior:
The app should display the input fields for entering the user's credentials when signing in or creating a new account. Credentials should be handled properly to successfully sign the user in or create a new account. 

**Device Model**: Pixel 6 API 29
**Android Version**: Android 10.0
**App version**: v7.1.1

### Duplicate Check
The bug has been discovered and is labelled [open](https://github.com/Docile-Alligator/Infinity-For-Reddit/issues/1560) on Github.

**Comment by lo-vil** [comment]
# Search Bar Bug: Quotation Marks

## Description

Typing " (quotation mark) in the search causes the app to crash, even though quotation marks can be added to recipe titles and descriptions.

## How to reproduce:
1. Launch Brocoli App
2. Click search icon
3. Type quotation mark (")

## Actual Behaviour
App Crashes

![Screenshot 2024-04-06 at 5 19 31 PM](https://github.com/SOEN345-WINTER2024/discussion/assets/94203138/ef5b83fc-dfab-4835-9db6-57038eb1a16f)

## Expected Behaviour
Simply put app should not crash. More specifically, if a recipe contains a quotation mark in its title, description, or url, then it should be queried and displayed in the recipe list. (Similar to if I type Chicken and the recipe Chicken Pasta recipe is displayed)

![Screenshot 2024-04-06 at 5 31 04 PM](https://github.com/SOEN345-WINTER2024/discussion/assets/94203138/ececba22-48d5-4d91-a341-cb1209269f76)

**Device Model:** Pixel 3a API 34
**Android Version:** Android 14.0
**App version:** v1.2.8-fdroid

## Duplicate Check
The issue has never been reported before.

**Link to issue reported:** [issue](https://github.com/flauschtrud/broccoli/issues/213)

**Comment by Fetyali7** [comment]
# Reorder Pages: Selecting a file from the recent section
### Description: 
When you select a file from the recent section to reorder the pages, you get an error saying that the file is encrypted or in the external storage.

### Steps to reproduce:

1. Launch the app Images-to-PDF
2. Navigate to 'Reorder Pages'
3. Click on 'Select A File'
4. Select a file from the recent section

### Expected behaviour:

The selected file should be imported.

### Actual behaviour:

Receive an error saying: We were unable to access PDF.  Files maybe encrypted or in the external storage. <br>
![Screenshot 2024-04-08 at 5 05 34 PM](https://github.com/Swati4star/Images-to-PDF/assets/70493307/c485fa65-1d8f-4774-b2ea-9b726be9f56a)


Device Model: Pixel 3a API 34
Android Version: Android 11.0
App version: v9.0.1

## Duplicate check:
The issue has never been reported before. <br>
Link to issue reported: https://github.com/Swati4star/Images-to-PDF/issues/1164

**Comment by jcjc1233** [comment]
# Bug Report: android-money-manager-ex

## Description
In the app, when attempting to use a specific currency, even after updating currencies, the app will not search for or display new currencies. It will also delete all the other currencies when you choose one of them for the first time

## Environment
- **Device Info**: Samsung S22 Ultra
- **android-money-manager-ex Version**: 2024.02.15 (1019)
- **Android Version**: 14
- **Stacktrace**: Bug does not cause a crash
- **Is it a Duplicate?**: No

## Steps to Reproduce
1. Open App
2. Click “hamburger” icon
3. Click on Entities
4. Click on Currencies
5. Click on the download button
6. Observe that no currencies will be downloaded
7. Attempt to search and use any new currency fails (example: US dollar or USD)

## Expected Behaviour
The app is supposed to show a list of downloaded currencies to choose from. Initially, you can select a currency before downloading, but once the download button is pressed, all other currencies are deleted and cannot be used.

## Actual Behaviour
After pressing the download button, the list of currencies is either not updated or is cleared, preventing the selection and use of new currencies.

![WhatsApp Image 2024-04-08 at 23 00 26_0b4ae5cb](https://github.com/moneymanagerex/android-money-manager-ex/assets/98419894/da87c326-4237-40c3-a63c-c879e0fc261d)

Link to issue reported https://github.com/moneymanagerex/android-money-manager-ex/issues/1653
